ui/components/app/signature-request/signature-request.container.js

```diff
@@ -60,12 +86,62 @@ function mapStateToProps(state, ownProps) {
     // not forwarded to component
     allAccounts: accountsWithSendEtherInfoSelector(state),
     ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    accountType: getAccountType(state),
+    isNotification: envType === ENVIRONMENT_TYPE_NOTIFICATION,
     selectedAccount: getSelectedAccount(state),
     ///: END:ONLY_INCLUDE_IN
   };
 }
 
-function mapDispatchToProps(dispatch) {
+let mapDispatchToProps = null;
+
+///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+function mmiMapDispatchToProps(dispatch) {
```

Comment:
Why is the mapping completely different in MMI versus jus adding keys to the existing mapDispatchToProps in this build type?

Reply:
Yes, that's right. It was done to give more clarity to the other devs. Happy to change if needed